src/less/carousel/stories/carousel.stories.js

```diff
@@ -29,6 +29,35 @@ export const continuous = () => `
 </div>
 `;
 
+export const imageTreatment = () => `
+<div class="carousel">
+    <div class="carousel__container carousel__container--image-treatment">
+        <button class="carousel__control carousel__control--prev" aria-label="Previous Slide - Top Products">
+            <svg aria-hidden="true" class="icon icon--carousel-prev" focusable="false">
+                <use xlink:href="#icon-carousel-prev"></use>
+            </svg>
+        </button>
+        <div class="carousel__viewport carousel__viewport--mask">
+            <ul class="carousel__list carousel__list--default-demo">
```

Comment:
If we don't specify fixed heights does it work still fine? Just checking

Reply:
Yes it does 👍